tests/test_private_cli.py

```diff
@@ -251,6 +251,22 @@ def test_log_comparison_parallel(tmp_path: str, input_genomes_small: Path) -> No
     )
 
     fasta = list(input_genomes_small.glob("*.fna"))  # subset of folder (2)
+    pool = Pool(len(fasta))
```

Comment:
We changed the extensions so there is now only one *.fna, so should probably hard code the pool size larger.